src/Application.php

```diff
@@ -28,19 +28,6 @@
 class Application extends BaseApplication
 {
     /**
-     * Load all the application configuration and bootstrap logic.
-     *
-     * You can include any other files your application needs to bootstrap
-     * in a web context here.
-     *
-     * @return void
-     */
-    public function bootstrap()
-    {
-        require_once $this->configDir . '/bootstrap.php';
```

Comment:
This was on purpose. My hope was that it made how the framework was working more obvious. Perhaps we could remove the base class method?